eva_submission/eload_validation.py

```diff
@@ -92,6 +94,27 @@ def _validate_sample_names(self):
             })
         self.eload_cfg.set('validation', 'sample_check', 'pass', value=not overall_differences)
 
+    def _validate_genotype_aggregation(self):
+        for analysis_alias in self.eload_cfg.query('submission', 'analyses'):
+            aggregations = [
+                detect_vcf_aggregation(vcf_file)
+                for vcf_file in self.eload_cfg.query('submission', 'analyses', analysis_alias, 'vcf_files')
+            ]
+            errors = []
```

Comment:
Should be initialised outside the for-loop, if you want to report all analyses' errors in the end.

Reply:
Good catch thank you